src/main/java/com/Payload/ConfirmPasswordEntity.java

```diff
@@ -0,0 +1,40 @@
+package com.Payload;
+
+public class ConfirmPasswordEntity {
```

Comment:
why is it called an Entity but doesn't have an @Entity? Is this persisted?

Reply:
It's used to Receive the user new password. But not for persistence. I will change the name to ConfirmPasswordPayload.